csfieldguide/chapters/content/en/glossary/correlation.md

```diff
@@ -0,0 +1,4 @@
+# Correlation
+
+A mutual relationship or connection between two or more variables.
+Two things are said to be correlated if changing the value or state of one also changes the value or state of the other.
```

Comment:
This describes *causation*, correlation is when two unrelated things happen to follow a similar trend

Reply:
I don't think so, the definition of correlation from Miriam Webster is 

> a relation existing between phenomena or things or between mathematical or statistical variables which tend to vary, be associated, or occur together in a way not expected on the basis of chance alone

Reply:
Okay, but `changing the value or state of one also changes the value or state of the other.` is still causation:

>  the act or agency which produces an effect 

You need to say that A and B change together, instead of saying that changing A changes B